src/components/ModelTableList/CloudGroup.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import { mount } from "enzyme";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+
+import CloudGroup from "./CloudGroup";
+
+import dataDump from "../../testing/complete-redux-store-dump";
+
+const mockStore = configureStore([]);
+
+describe("CloudGroup", () => {
+  it("renders without crashing and matches snapshot", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <MemoryRouter>
+        <Provider store={store}>
+          <CloudGroup />
+        </Provider>
+      </MemoryRouter>
+    );
+    expect(wrapper.find(CloudGroup)).toMatchSnapshot();
```

Comment:
This test is not very useful as it creates a 5k loc file that nobody can actually review. I'd propose just removing it as it'll fail on the next test if it can't render.